src/api/loader.coffee

```diff
@@ -13,15 +13,19 @@ getAjaxSettingsByEnv = (isLocal, baseUrl, setting, eventData) ->
 
   {data, change} = eventData
   apiSetting = _.pick(setting, 'url', 'method')
+  apiSetting.dataType = 'json'
+  apiSetting.contentType = 'application/json;charset=UTF-8'
+
   if _.includes(METHODS_WITH_DATA, apiSetting.method)
-    apiSetting.data = change or data
+    apiSetting.data = JSON.stringify(change or data)
 
```

Comment:
jQuery would handle the serialization of an object as json internally, but doesn't hurt to be explicit.